main.go

```diff
@@ -147,7 +147,7 @@ func gungnir(arguments []string) {
 	exitIfError(logger, emperror.Wrap(err, "failed to initialize cipher config"))
 	decrypters := voynicrypto.PopulateCiphers(cipherOptions, logger)
 
-	gungnirHandler, err := authChain(config.AuthHeader, config.JwtValidator, config.CapabilityCheck, logger, metricsRegistry)
+	gungnirHandler, err := authChain(v, config.AuthHeader, config.CapabilityCheck, logger, metricsRegistry)
```

Comment:
Can we avoid passing the viper object in and instead pass in the needed configuration?

Reply:
totally

Reply:
done